relay-general/src/processor/traits.rs

```diff
@@ -107,7 +110,14 @@ pub trait Processor: Sized {
 }
 
 /// A recursively processable value.
-pub trait ProcessValue: FromValue + ToValue + Debug {
+pub trait ProcessValue:
+    FromValue
+    + ToValue
+    + Debug
+    + Attributes<PiiAttrs>
```

Comment:
I could not find a nice way to make ProcessValue be completely generic over the type of attributes it deals with, so this is what I ended up with. The problem I ran into was related to processors who do not want to know about attributes at all.

I think this splitup still brings us a bit closer.